src/main/java/com/bunq/sdk/http/BunqHeader.java

```diff
@@ -0,0 +1,100 @@
+package com.bunq.sdk.http;
+
+import java.util.Collection;
+import java.util.Map;
+
+public enum BunqHeader {
+    attachmentDescription("X-Bunq-Attachment-Description"),
+    cacheControl("Cache-Control", "no-cache"),
+    contentType("Content-Type"),
+    clientAuthentication("X-Bunq-Client-Authentication"),
```

Comment:
For better read and usability, perpend `BUNQ_`  to all the bunq headers. `isBunq()` should stay nevertheless.